.github/workflows/pull-request.yaml

```diff
@@ -102,8 +102,12 @@ jobs:
           repository: open-policy-agent/npm-opa-wasm
           path: npm-opa-wasm
 
+      # NOTE(sr): the opa binary built above doesn't have wasm enabled,
+      # so we override the capabilities for what the released binary
+      # provides.
```

Comment:
~I'm not sure I follow this comment. Why does this job have a dependency on the wasm runtime in OPA?~ I see, because we have a build constraint on the `github.com/open-policy-agent/opa/internal/wasm/sdk/opa/capabilities` package we need this.